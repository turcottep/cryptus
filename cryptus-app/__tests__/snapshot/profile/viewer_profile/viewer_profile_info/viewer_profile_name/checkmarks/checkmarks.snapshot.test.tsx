// components/__tests__/button.snapshot.test.tsx
import React from "react";
import renderer from "react-test-renderer";
import Checkmarks from "../../../../../../../components/profile/viewer_profile/viewer_profile_infos/viewer_profile_name/checkmarks/checkmarks";

it("Checkmarks renders correctly", () => {
  const tree = renderer.create(<Checkmarks />).toJSON();
  expect(tree).toMatchSnapshot();
});
