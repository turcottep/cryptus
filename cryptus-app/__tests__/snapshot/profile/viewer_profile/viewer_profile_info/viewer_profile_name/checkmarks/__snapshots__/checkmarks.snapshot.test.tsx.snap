// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkmarks renders correctly 1`] = `
<div>
  <img
    src="./images/twitter_checkmark.png"
  />
</div>
`;
